search-app/docs/api.md

```diff
@@ -2447,7 +2447,23 @@ For granule additional attributes search, the default is searching for the attri
 
 #### <a name="g-spatial"></a> Find granules by Spatial
 The parameters used for searching granules by spatial are the same as the spatial parameters used in collections searches. (See under "Find collections by Spatial" for more details.)
+
 Note: When querying a granule which has multiple types of spatial features in the granule metadata (i.e. a Polygon and a Bounding Box), the granule will be returned if the spatial query matches at least one of the spatial types on the given granule (i.e. matches the granule's Polygon OR Bounding Box).
+This may not always be the desired behavior if a granule has a bounding rectangle which is less precise than the polygons. For this, you may use the 'ignore_br' or 'every' keywords when doing spatial searches. These flags can be applied to any type of spatial search (polygon, bounding box, circle, line, point).
+
+The 'ignore_br' flag will ignore the bounding rectangles of the granule and look to match at least one other geometry (i.e. one or more of the polygons).
+
+    curl "%CMR-ENDPOINT%/granules?collection_concept_id=%CMR-EXAMPLE-COLLECTION-ID%&polygon[ignore_br]=10,10,30,10,30,20,10,20,10,10"
```

Comment:
what if there are multiple spatial areas in the search parameter? I think a better way to pass in the spatial search `ignore_br` is via the options, i.e. `options[spatial][ignore_br]=true`.

Reply:
If there are multiple spatial areas in the search parameter, the flag can currently be applied to each spatial area:

curl "http://localhost:3003/collections?polygon[ignore_br]=10,10,30,10,30,20,10,20,10,10&polygon[every]=11,11,31,11,31,21,11,21,11,11"

Passing it via options would mean all spatial areas would be restricted to the same behavior. I could still make the change if you prefer, but it would require _significant_ code change at this point to move it to options.

Reply:
I know it will require significant code change to switch to `options` parameter. It is unfortunate that we didn't settle on the best API before starting implementation. I still think the `options` parameter makes more sense than individual spatial parameters. I just don't see the use case of the example search that you give above. I don't even see the use case of the `every` option. I don't want to force you to rewrite everything. I would say bring this up with the team and get other people's opinions.

Reply:
Made the suggested change to API interaction to be of the format `options[spatial][ignore_br]=true`